src/Spinner.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+import './Spinner.css';
+
+function Spinner() {
+  return (
+    <Fragment>
+      <div className="lds-spinner"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
```

Comment:
this is too hacky. I'd just use one of these icons: https://icons8.com/icons/set/spinner and have a basic spinning animation: http://jsfiddle.net/gionaf/Ugc5g/ (you only need the keyframes bit)